falcon.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env falcon host
+# frozen_string_literal: true
+
+require 'falcon/environment/rack'
+require 'falcon/environment/supervisor'
+require '3scale/backend/manifest'
+
+HOSTNAME = 'apisonator_listener'
```

Comment:
`_` is not a valid domain name character

Reply:
https://github.com/3scale/apisonator/pull/427/commits/4b12b83e18f1522db9a24491ee9994637db79bdc